training/templates/participant_transaction_list_new.html

```diff
@@ -155,7 +155,11 @@
             <td>{% with event=record.paymentdetail.purpose|get_event_details %}
               
                 {{event.event_name}} <br><br>
-                <b>{{event.foss.foss}}</b><br>
+                <b>
+                  {% for foss in event.course.foss.all %}
```

Comment:
```suggestion
                  {% for foss in event.course.foss.all() %}
```
Does this require brackets?